javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,6 @@
+const yearOfBirth = 1992;
+let yearFuture = 2045;
```

Comment:
use const unless you want to re-assign the value of variable.

Reply:
thank you kavya, for your review on my code . :)